src/cascadia/TerminalSettingsEditor/ProfileViewModel.h

```diff
@@ -140,6 +163,47 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
         Model::CascadiaSettings _appSettings;
         Editor::AppearanceViewModel _unfocusedAppearanceViewModel;
+
+        enum class PaddingDirection
+        {
+            Left = 0,
+            Top = 1,
+            Right = 2,
+            Bottom = 3
+        };
+
+        winrt::hstring _GetNewPadding(PaddingDirection paddingDirection, double newPaddingValue)
+        {
+            std::wstringstream result;
+            std::array<double, 4> values{};
+            std::wstring_view remaining{ Padding() };
+            uint32_t paddingIndex = static_cast<uint32_t>(paddingDirection);
+
+            try
+            {
+                for (uint32_t index = 0; !remaining.empty() && index < values.size(); ++index)
+                {
+                    const std::wstring token{ til::prefix_split(remaining, L',') };
+                    auto curVal = std::stod(token);
+
+                    if (paddingIndex == index)
+                    {
+                        curVal = newPaddingValue;
+                    }
+
+                    values[index] = curVal;
+                }
+            }
+            catch (...)
+            {
+                values.fill(0);
+                LOG_CAUGHT_EXCEPTION();
+            }
+
+            result << values[0] << L", " << values[1] << L", " << values[2] << L", " << values[3];

```

Comment:
We use `fmt` instead of `sstream` almost everywhere. This would also allow us to easily round the padding to e.g. 6 digits during formatting, just in case (to avoid `0.3000000000000001` situations).

Something like this:
```cpp
const auto result = fmt::format(
    FMT_COMPILE(L"{:.6f},{:.6f},{:.6f},{:.6f}"),
    values[0],
    values[1],
    values[2],
    values[3]
);
return winrt::hstring{ result };
```

Theoretically you should also have access to `fmt::join` so this may work:
```cpp
const auto result = fmt::format(FMT_COMPILE(L"{:.6f}"), fmt::join(values, L","));
```